src/coreclr/tools/Common/TypeSystem/Common/Utilities/GCPointerMap.Algorithm.cs

```diff
@@ -101,7 +101,7 @@ public static GCPointerMap FromStaticLayout(MetadataType type)
             return builder.ToGCMap();
         }
 
-        private static void MapThreadStaticsForType(GCPointerMapBuilder builder, MetadataType type, int baseOffset)
+        private static void MapThreadStaticsForType(ref GCPointerMapBuilder builder, MetadataType type, int baseOffset)
```

Comment:
The `ref` part is the follow up change that somehow missed the previous PR.
https://github.com/dotnet/runtime/pull/84566#discussion_r1185370998